### ISSUE TITLE
Lack of libffi causes some Python packages to fail to be installed.

### ISSUE THREAD

**Issue by GrahamDumpleton** [sclorg/s2i-python-container#122]
This appears to affect at least `ndg-httpsclient` a dependency required by Lektor site generator.

```
Running setup.py install for ndg-httpsclient
Skipping installation of /opt/app-root/src/.local/lib/python3.4/site-packages/ndg/__init__.py (namespace package)

Installing /opt/app-root/src/.local/lib/python3.4/site-packages/ndg_httpsclient-0.4.1-py3.4-nspkg.pth
Installing ndg_httpclient script to /opt/app-root/src/.local/bin
Running setup.py install for cryptography
Package libffi was not found in the pkg-config search path.
Perhaps you should add the directory containing `libffi.pc'
to the PKG_CONFIG_PATH environment variable
No package 'libffi' found
Package libffi was not found in the pkg-config search path.
Perhaps you should add the directory containing `libffi.pc'
to the PKG_CONFIG_PATH environment variable
No package 'libffi' found
Package libffi was not found in the pkg-config search path.
Perhaps you should add the directory containing `libffi.pc'
to the PKG_CONFIG_PATH environment variable
No package 'libffi' found
Package libffi was not found in the pkg-config search path.
Perhaps you should add the directory containing `libffi.pc'
to the PKG_CONFIG_PATH environment variable
No package 'libffi' found
Package libffi was not found in the pkg-config search path.
Perhaps you should add the directory containing `libffi.pc'
to the PKG_CONFIG_PATH environment variable
No package 'libffi' found
c/_cffi_backend.c:15:17: fatal error: ffi.h: No such file or directory
#include <ffi.h>
^
compilation terminated.
Traceback (most recent call last):
File "/opt/rh/rh-python34/root/usr/lib64/python3.4/distutils/unixccompiler.py", line 126, in _compile
extra_postargs)
File "/opt/rh/rh-python34/root/usr/lib64/python3.4/distutils/ccompiler.py", line 909, in spawn
spawn(cmd, dry_run=self.dry_run)
File "/opt/rh/rh-python34/root/usr/lib64/python3.4/distutils/spawn.py", line 36, in spawn
_spawn_posix(cmd, search_path, dry_run=dry_run)
File "/opt/rh/rh-python34/root/usr/lib64/python3.4/distutils/spawn.py", line 162, in _spawn_posix
% (cmd, exit_status))
distutils.errors.DistutilsExecError: command 'gcc' failed with exit status 1

```

**Comment by GrahamDumpleton** [comment]
Package for this one is 'libffi-devel'.

**Comment by pkubatrh** [comment]
Closed via #175
